recipes/salib/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: salib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/SALib/SALib
+  git_tag: v{{ version }}
```

Comment:
According to the [docs]( http://pyscaffold.readthedocs.io/en/v2.5.7/features.html#complete-git-integration ) it say `pyscaffold` uses `setuptools_scm`. It is possible to override the version used by `setuptools_scm` with the environment variable `SETUPTOOLS_SCM_PRETEND_VERSION` as mentioned in their [README]( https://github.com/pypa/setuptools_scm/tree/39aa9126bafec549820b6ccb2161c889d7e68ebe#environment-variables ).